packages/deployments/contracts/contracts/core/connext/libraries/AmplificationUtils.sol

```diff
@@ -58,10 +55,10 @@ library AmplificationUtils {
       uint256 a0 = self.initialA; // initial A value when ramp is started
       if (a1 > a0) {
         // a0 + (a1 - a0) * (block.timestamp - t0) / (t1 - t0)
-        return a0.add(a1.sub(a0).mul(block.timestamp.sub(t0)).div(t1.sub(t0)));
+        return a0 + ((a1 - a0) * (block.timestamp - t0)) / (t1 - t0);
       } else {
         // a0 - (a0 - a1) * (block.timestamp - t0) / (t1 - t0)
-        return a0.sub(a0.sub(a1).mul(block.timestamp.sub(t0)).div(t1.sub(t0)));
+        return a0 - ((a0 - a1) * (block.timestamp - t0)) / (t1 - t0);
```

Comment:
Based on https://docs.soliditylang.org/en/v0.8.15/cheatsheet.html?#order-of-precedence-of-operators isn't `((a0 - a1) * (block.timestamp - t0)) / (t1 - t0);` evaluated first. Should this be `(a0 + ((a0 - a1) * (block.timestamp - t0))) / (t1 - t0);`?

Reply:
Just want to confirm that its intended that `a0` is added to the result of `((a0 - a1) * (block.timestamp - t0)) / (t1 - t0);` and not added to the numerator before division.

Reply:
https://github.com/curvefi/curve-contract/blob/b0bbf77f8f93c9c5f4e415bce9cd71f0cdee960e/contracts/pools/3pool/StableSwap3Pool.vy#L160

I believe that is correct.

Reply:
cool, that looks consistent then.